### ISSUE TITLE
Manuscript file upload progress "stuck" at 0% 

### ISSUE THREAD

**Issue by chris-huggins** [elifesciences/elife-xpub#2249]
**Describe the bug**
When uploading a manuscript file on reviewer.elifesciences.org it shows at 0% file upload progress (i.e. doesn't count up). After refreshing a few times it shows as uploaded so assume client side is not showing progress despite upload "working".

**Steps To Reproduce**
Upload a manuscript file!

**Expected behavior**
Client will count up to 100% to feedback upload

**Device, Operating System and Browser**
Chrome 75 on Mac OSX Mojave

**Comment by hdrury1** [comment]
I'm able to reproduce this bug in Brave on Mac Mojave.

Let's timebox an investigation into this to find out how big a piece of work it is to fix.

**Comment by Cooryd** [comment]
subscription on the client side is receiving:

```
message: "Cannot use GraphQLSchema "[object GraphQLSchema]" from another module or realm.↵↵Ensure that there is only one instance of "graphql" in the node_modules↵directory. If different versions of "graphql" are the dependencies of other↵relied on modules, use "resolutions" to ensure only one version is installed.↵↵https://yarnpkg.com/en/docs/selective-version-resolutions↵↵Duplicate "graphql" modules cannot be used at the same time since different↵versions may have different capabilities and behavior. The data from one↵version used in the function from another could produce confusing and↵spurious results."
```

When trying to connect

**Comment by Cooryd** [comment]
https://github.com/elifesciences/elife-xpub/commit/0222222d3f4bea2bc0a3c79a662042d053436e57 works

<img width="1680" alt="Screenshot 2019-07-09 at 14 23 48" src="https://user-images.githubusercontent.com/22455561/60891461-47152a00-a255-11e9-8580-da8c59a6b5be.png">

https://github.com/elifesciences/elife-xpub/commit/a74df771e77cd9595cdd2821428a7498dd25ea99 breaks

<img width="1675" alt="Screenshot 2019-07-09 at 14 20 52" src="https://user-images.githubusercontent.com/22455561/60891248-dd951b80-a254-11e9-9794-317038c48992.png">

Looks like the pubsweet upgrade has affected the apollo subscription, likely a dependency change.

**Comment by erezmus** [comment]
Running `npm ls graphql` gives the following clues:

```
├─┬ pubsweet-client@8.0.0
│ ├─┬ apollo-link@1.2.2
│ │ └── graphql@14.4.2  extraneous
│ ├─┬ apollo-link-context@1.0.8
│ │ └─┬ apollo-link@1.2.3
│ │   └── graphql@14.0.2  extraneous
│ └─┬ apollo-link-http@1.5.4
│   ├─┬ apollo-link@1.2.3
│   │ └─┬ apollo-utilities@1.3.2
│   │   └── graphql@14.4.2  extraneous
│   └─┬ apollo-link-http-common@0.2.4
│     └─┬ apollo-link@1.2.3
│       └─┬ apollo-utilities@1.3.2
│         └── graphql@14.4.2  extraneous
└─┬ pubsweet-server@13.5.0
  ├─┬ apollo-server-express@2.6.4
  │ └─┬ apollo-server-core@2.6.4
  │   └─┬ apollo-engine-reporting@1.3.2
  │     ├─┬ apollo-graphql@0.3.2
  │     │ └── graphql@14.3.1  extraneous
  │     ├─┬ apollo-server-core@2.6.4
  │     │ ├── graphql@14.3.1  extraneous
  │     │ ├─┬ graphql-subscriptions@1.1.0
  │     │ │ └── graphql@14.3.1  extraneous
  │     │ └─┬ subscriptions-transport-ws@0.9.16
  │     │   └── graphql@14.0.2  extraneous
  │     └── graphql@14.3.1  extraneous
  ├── graphql@14.3.1 
  └─┬ subscriptions-transport-ws@0.9.16
    └── graphql@14.0.2  extraneous

npm ERR! extraneous: graphql@14.0.2 /Users/erez/Development/elife-xpub/node_modules/apollo-link/node_modules/graphql
npm ERR! extraneous: graphql@14.4.2 /Users/erez/Development/elife-xpub/node_modules/pubsweet-client/node_modules/apollo-link/node_modules/graphql
npm ERR! extraneous: graphql@14.4.2 /Users/erez/Development/elife-xpub/node_modules/pubsweet-client/node_modules/apollo-link-http-common/node_modules/apollo-utilities/node_modules/graphql
npm ERR! extraneous: graphql@14.4.2 /Users/erez/Development/elife-xpub/node_modules/pubsweet-client/node_modules/apollo-link-http/node_modules/apollo-utilities/node_modules/graphql
npm ERR! extraneous: graphql@14.3.1 /Users/erez/Development/elife-xpub/node_modules/apollo-engine-reporting/node_modules/graphql
npm ERR! extraneous: graphql@14.3.1 /Users/erez/Development/elife-xpub/node_modules/apollo-graphql/node_modules/graphql
npm ERR! extraneous: graphql@14.3.1 /Users/erez/Development/elife-xpub/node_modules/apollo-server-core/node_modules/graphql
npm ERR! extraneous: graphql@14.3.1 /Users/erez/Development/elife-xpub/node_modules/graphql-subscriptions/node_modules/graphql
npm ERR! extraneous: graphql@14.0.2 /Users/erez/Development/elife-xpub/node_modules/subscriptions-transport-ws/node_modules/graphql
npm ERR! extraneous: graphql@14.0.2 /Users/erez/Development/elife-xpub/node_modules/pubsweet-server/node_modules/subscriptions-transport-ws/node_modules/graphql
```

One solution is to use resolutions:
https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

**Comment by Cooryd** [comment]
Adding `"pubsweet-client/**/graphql": "14.4.2"` to the resolutions in webpack config seems to drop solve the extraneous flag on running `npm ls graphql` but doesn't solve the error on the subscription.

**Comment by Cooryd** [comment]
Next step of the investigation:

Compare the `graphql` versions included in https://github.com/elifesciences/elife-xpub/commit/0222222d3f4bea2bc0a3c79a662042d053436e57 and see whats different with current `develop` branch.

`yarn list graphql` 

022222 - 
└─ graphql@14.0.2
✨  Done in 1.00s.

develop - 
├─ graphql@14.0.2
└─ pubsweet-server@13.5.0
   └─ graphql@14.3.1
✨  Done in 1.04s.

🤔 could the servers separate graphql dep be getting bundled by webpack unintentionally

On adding `graphql@14.3.1` as a dependency to the project, the list tree now looks like this: 

├─ graphql@14.3.1
└─ pubsweet-client@8.0.0
   └─ graphql@14.0.2

Looks like client is fixed to needing 14.0.2 and server fixed to needing 14.3.1

**Comment by Cooryd** [comment]
Solution 1:

`yarn add graphql@14.3.1`

then add `"pubsweet-client/graphql": "14.3.1"` to `resolutions` in `package.json`

Will investigate if there's a less brute force way of doing this.
